llama/llama.go

```diff
@@ -1,8 +1,8 @@
 package llama
 
 /*
-#cgo CFLAGS: -std=c11 -DNDEBUG -DLOG_DISABLE_LOGS -DGGML_USE_LLAMAFILE
-#cgo CXXFLAGS: -std=c++11 -DNDEBUG -DLOG_DISABLE_LOGS -DGGML_USE_LLAMAFILE
+#cgo CFLAGS: -O2 -std=c11 -DGGML_BUILD=1 -DNDEBUG -DLOG_DISABLE_LOGS -DGGML_USE_LLAMAFILE
+#cgo CXXFLAGS: -O2 -std=c++11 -DGGML_BUILD=1 -DNDEBUG -DLOG_DISABLE_LOGS -DGGML_USE_LLAMAFILE
```

Comment:
I think you had this as -O3 before but I guess -O2 is the correct one? Is there any benefit from -O3 instead?

Reply:
The generate/cmake uses -O2, so this is more correct than my prior change.